config/default.yaml

```diff
@@ -36,3 +37,23 @@ modules:
     # - ./modules/statsd
     # - ./modules/openTSDB
     # - ./modules/influxdb
+
+  whitelistedKeys:
```

Comment:
In trying to run this, it looks like this file is invalid YAML in this current form.  I think you need to fix the indentation perhaps?